packages/grafana-data/package.json

```diff
@@ -7,18 +7,30 @@
   "keywords": [
     "typescript"
   ],
+  "sideEffects": false,
   "repository": {
     "type": "git",
     "url": "http://github.com/grafana/grafana.git",
     "directory": "packages/grafana-data"
   },
   "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "main": "dist/index.js",
+    "module": "dist/esm/index.js",
+    "types": "dist/index.d.ts",
+    "access": "public"
+  },
+  "files": [
+    "dist",
+    "CHANGELOG.md",
+    "LICENSE_APACHE2"
+  ],
   "scripts": {
-    "build": "grafana-toolkit package:build --scope=data",
-    "bundle": "rollup -c rollup.config.ts",
+    "build": "tsc -p ./tsconfig.build.json && rollup -c rollup.config.ts",
     "clean": "rimraf ./dist ./compiled",
     "docsExtract": "mkdir -p ../../reports/docs && api-extractor run 2>&1 | tee ../../reports/docs/$(basename $(pwd)).log",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --emitDeclarationOnly false --noEmit"
```

Comment:
I see now, it needs to be set here because the compiler defaults are overriden in the package tsconfig files. 👍 
(I would have thought that `--noEmit` stops anything from being emitted, but I guess that's not how tsc works)

Reply:
Unfortunately you cannot set `--emitDeclarationOnly true --noEmit true` the cli throws an error and bails. There's an open issue [here](https://github.com/microsoft/TypeScript/issues/32380) about it.

Reply:
I see, makes sense then, thanks!